tutorials/upgrade-managed-postgresql-database/index.mdx

```diff
@@ -0,0 +1,412 @@
+---
+title: Upgrading your Managed Database for PostgreSQL to the latest version
+description: This page shows how to upgrade your Managed Database for PostgreSQL to the latest version 
+tags: storage, postgresql, database
+excerpt: |
+  PostgreSQL is a powerful, open-source object-relational database system with more than 30 years of active development. As major PostgreSQL releases regularly add new features, the layout of the system tables may change. However the internal data storage format itself rarely changes. This allows you to upgrade a PostgreSQL database system without data loss using the backup and restore functions of the Database API.
+categories:
+  - storage
+  - database
+image: /assets/images/docs/scaleway.png
+permalink: docs/how-to-upgrade-managed-database-for-postgresql/
+keywords: cloud tutorials, cloud documentations, cloud instance, cloud hosting, server, storage, database, postgresql, sql, database engine, postgre
+---
+
+<Message type="important">
+
+PostgreSQL version 9.6 reaches its End of Life (EoL) on November 11, 2021. It is **recommended** to upgrade your PostgreSQL database engine to the latest available version before this date.
+
+</Message>
+
+[PostgreSQL](https://www.postgresql.org/) is a powerful, open-source object-relational database system with more than 30 years of active development. As major PostgreSQL releases regularly add new features, the layout of the system tables may change. However, the internal data storage format itself rarely changes. This allows you to upgrade a PostgreSQL database system without data loss using the backup and restore functions of the Database API.
+
+In this tutorial, you will learn how to migrate your PostgreSQL databases from version 9.6 to 13. This can be achieved in two ways:
+
+- Manually, using the console and the API. This could be a good option if you only have one database to migrate.
+- Automatically, using a bash script based on the Scaleway CLIv2. This could be a good option if you have multiple databases to migrate, and/or are concerned about 
+errors during a manual restore.
+
+
+## Manual migration via the Scaleway console and API
```

Comment:
Could you add a warning on this method as it will fail if client got extensions installed on its instances. 

We recommand to use the script method or a manual dump/restore for instances with extensions i.e timescaledb, postgis ...